package/MDAnalysis/core/universe.py

```diff
@@ -1459,13 +1488,98 @@ def from_smiles(cls, smiles, sanitize=True, addHs=True,
                 "hydrogens with `addHs=True`")
 
             numConfs = rdkit_kwargs.pop("numConfs", numConfs)
-            if not (type(numConfs) is int and numConfs > 0):
+            if not (isinstance(numConfs, int) and numConfs > 0):
                 raise SyntaxError("numConfs must be a non-zero positive "
-                "integer instead of {0}".format(numConfs))
+                                  "integer instead of {0}".format(numConfs))
             AllChem.EmbedMultipleConfs(mol, numConfs, **rdkit_kwargs)
 
         return cls(mol, **kwargs)
 
+    def guess_TopologyAttributes(
```

Comment:
```suggestion
    def guess_TopologyAttrs(
```

This is just a suggestion and I'm very open to being persuaded otherwise, but at the moment I would very much prefer that this method is called `guess_TopologyAttrs` instead. In all other contexts our topology attributes are called `TopologyAttr`, e.g. the base `topologyattrs.TopologyAttr` class, and existing methods follow this theme such as `add_TopologyAttr` and `del_TopologyAttr`.

@orbeckst I think you advised switching to `guess_TopologyAttributes` in an earlier review -- do you maybe have another perspective on the naming here?

Reply:
> This is just a suggestion and I'm very open to being persuaded otherwise, but at the moment I would very much prefer that this method is called `guess_TopologyAttrs` instead. In all other contexts our topology attributes are called `TopologyAttr`, e.g. the base `topologyattrs.TopologyAttr` class, and existing methods follow this theme such as `add_TopologyAttr` and `del_TopologyAttr`.
> 
> @orbeckst I think you advised switching to `guess_TopologyAttributes` in an earlier review -- do you maybe have another perspective on the naming here?

I'm open to both names. However, I'm more convinced with your point of view

Reply:
Great! Just pinging @orbeckst again in case he has a different opinion :-)

Reply:
Since @orbeckst hasn't checked back on this, would you mind making this change, @aya9aladdin? It makes for a much more consistent interface.